integration-test/test/handler-commands.bats

```diff
@@ -21,6 +21,16 @@ teardown(){
     [[ "$diff" = *"A /var/lib/azure/custom-script"* ]]
```

Comment:
Have you considered moving this to under/var/lib/waagent/<something>/custom-script, or a file under /var/lib/waagent/custom-script.<something>.